src/python/WMComponent/JobSubmitter/JobSubmitterPoller.py

```diff
@@ -363,6 +362,11 @@ def refreshCache(self):
                 numberOfCores = getattr(baggage, "numberOfCores", 1)
             loadedJob['numberOfCores'] = numberOfCores
 
+            # check if job should be submitted as highIO job
+            highIOjob = False
+            if newJob['type'] in ["Repack", "Merge", "Cleanup", "LogCollect"]:
```

Comment:
Was thinking about that but decided against it. They really don't use much IO and can use non-negligible cpu, so I decided they are better off as standard single core jobs.